specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -9945,7 +9948,7 @@
           "Node get remote login settings": { "$ref": "./examples/NodeGetRemoteLoginSettings.json" }
         },
         "summary": "Gets the settings required for remote login to a compute node.",
-        "description": "Before you can remotely login to a node using the remote login settings, you must create a user account on the node. This API can be invoked only on pools created with the virtual machine configuration property.",
+        "description": "Before you can remotely login to a node using the remote login settings, you must create a user account on the node. This API can be invoked only on pools created with the virtual machine configuration property. For pools created with a cloud service configuration, see the GetRemoteDesktop API.",
```

Comment:
>GetRemoteDesktop [](start = 298, length = 16)

Probably ought to change this to the MSDNified name e.g. "the 'connect to a compute node using Remote Desktop Protocol' API" or whatever they call it